src/ed/entity.py

```diff
@@ -67,12 +67,13 @@ def __init__(
     def volumes(self):
         return self._volumes
 
-    def in_volume(self, point: VectorStamped, volume_id: str) -> bool:
+    def in_volume(self, point: VectorStamped, volume_id: str, padding: float = 0) -> bool:
```

Comment:
Python is going to interpret 0 as an int, I would change it to 0.0

Reply:
For a default value of zero it doesn't matter. And as it is only added or substracted. It doesn't matter either.

Reply:
I know, there is no problem with the program. I said that so that it complies with the specification of  ```padding: float = 0```

Reply:
I think `0` is totally fine.